### DISCUSSION TITLE
Is there any way to get json data from excalidraw using javascript?

### DISCUSSION THREAD

**Discussion by GurdeepSingh55** [excalidraw/excalidraw#7014] [Q&A]
I am trying to use excalidraw in .net core project. I am able to load excalidraw. I am facing issue when i am trying to get  the scene elements from excalidraw using javascript

**Comment by ghost** [comment]
find a way to call this method from javascript 
<img width="506" alt="image" src="https://github.com/excalidraw/excalidraw/assets/6542274/fcfaf9c8-46ad-4cf7-be24-463f9fc3aef3">
